qml/MainContent.qml

```diff
@@ -84,11 +97,49 @@ Item {
                 }
             }
         }
+
+        // current interval
+        RowLayout {
+            visible: controller.settings.cyclesMode
+
+            spacing: Style.smallSpacing
+
+            CustomLabel {
+                text: qsTr("Current interval:")
+            }
+            EditableIntegerLabel {
```

Comment:
![resto1](https://user-images.githubusercontent.com/25504382/109304338-6868c680-783c-11eb-83f9-bd9c8a74f306.png)

Reply:
Btw. when you're showing `Current interval` it would be nice to have some graphics instead of raw number. Maybe apples if not tomatoes :apple:

Reply:
>  it would be nice to have some graphics

The idea is nice in a concept but I think it wouldn't match the rest of UI! 😉 Thanks for your suggestion!

Reply:
I must admit that the whole UI looks a little bit strange on your OS.
Can you please test this again after my newest changes?
Thanks!

Reply:
The spin box is fixed. The small thing I see on my side is:

![meh](https://user-images.githubusercontent.com/25504382/111802806-b4081080-88ce-11eb-9387-c0d3cebdbbd9.png)